src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/JndiDirectoryDAO.scala

```diff
@@ -54,10 +54,23 @@ class JndiDirectoryDAO(protected val directoryConfig: DirectoryConfig)(implicit
     }
   }
 
+  // Forcibly delete a group regardless of its membership
+  // If you want to verify that the group is not a member of any other group prior to deletion, use safeDeleteGroup
   override def deleteGroup(groupName: WorkbenchGroupName): Future[Unit] = withContext { ctx =>
```

Comment:
Instead of `safeDeleteGroup`/`deleteGroup`, I'd prefer `deleteGroup`/`forceDeleteGroup`, making "safe" the default behavior. That would make it harder to accidentally discover and call the wrong function when working with a `DirectoryDAO`. I haven't looked at remaining callers of the current `deleteGroup` to know if that would make that code read funny. Thoughts?

Reply:
Agreed that this would make more sense.  I contemplated just having the single `deleteGroup` method and rewriting it to always be "safe", but was concerned about breaking code that I might not understand.  Now that I look at it though, up until this PR, the only code calling `DirectoryDAO.deleteGroup()` has been tests.  

Therefore, since this is the first and only production use case for deleting a group so far, I would suggest we just have the one method for `deleteGroup` that does it in the "safe" way, and if we need a "force" version later, we can implement it then. 

I am going to make this change now, which may take a little bit of time to make sure all the tests stay green.

Reply:
Fixed